oh-my-git.plugin.zsh

```diff
@@ -230,7 +231,7 @@ function oh_my_git_info {
 		
 			else
 				oh_my_git_string+="
-					${has_no_upstream_color}(${current_branch_color}${current_branch}${reset}${has_no_upstream_color})${reset}
+					${has_no_upstream_color}(${current_branch_color}${current_branch}${reset}${has_no_upstream_color}${has_no_upstream_symbol})${reset}
```

Comment:
Wouldn't it be better if it was placed at the same position as the upstream indicator ([line 190-192]](https://github.com/bric3/oh-my-zsh-git/pull/3/commits/0350b074b11ef89ac1ddd648054fb94517c3c8ad#diff-d65f4cbd76d7f4fbc68f321e1ae82c3fR190)). Also I think it should be displayed only if the same (`display_has_upstream`) is enabled, or a similar one.

Reply:
Yes, you're right, I didn't think of that. I've changed it.

I think the existing flag 'display_has_upstream' is sufficient for this.